index.html

```diff
@@ -55,8 +56,107 @@ <h1 class="hero-title">
           <img src="images/company-logos/microsoft-logo.png" alt="company logo">
           <img src="images/company-logos/walmart-logo.png" alt="company logo">
           <img src="images/company-logos/fedex-logo.png" alt="company logo">
-        </div>
-      </section>   
+          </div>
+      </section>
+      <!--feature-section-->
```

Comment:
Please ensure that your PRs are dedicated to one section only. 
`Hero-section` and `company-logos` shouldn't be included here.

- [ ] Remove the Hero and logo sections leaving only the features-related code.
> remember to remove it as well from the CSS file.